lm_eval/api/task.py

```diff
@@ -941,13 +941,13 @@ def fewshot_context(self, doc: str, num_fewshot: int) -> str:
             The fewshot context.
         """
 
+        description = utils.apply_template(self.config.description, doc)
```

Comment:
Thanks very much for the PR! any chance we could skip the jinja rendering if description is not provided?
```suggestion
	if self.config.description:
	        description = utils.apply_template(self.config.description, doc)
```